regulation-worker/internal/client/client.go

```diff
@@ -86,9 +88,9 @@ func (j *JobAPI) UpdateStatus(ctx context.Context, status model.JobStatus, jobID
 
 	method := "PATCH"
 
-	genEndPoint := "/worker/workspaces/{workspace_id}/regulations/worker-job/{job_id}"
+	genEndPoint := "/dataplane/workspaces/{workspace_id}/regulations/workerJobs/{job_id}"
 	url := fmt.Sprint(j.URLPrefix, prepURL(genEndPoint, j.WorkspaceID, fmt.Sprint(jobID)))
-	statusSchema := statusJobSchema{
+	statusSchema := StatusJobSchema{
```

Comment:
Is it exported just for main_test?

Reply:
yes, but will look for alternatives

Reply:
added schema_test.go in cmd for main_test.go